pkg/virt-api/api.go

```diff
@@ -277,10 +277,9 @@ func (app *virtAPIApp) composeSubresources() {
 			Operation("vnc").
 			Doc("Open a websocket connection to connect to VNC on the specified VirtualMachineInstance."))
 
+		// An empty handler function would respond with HTTP OK by default
 		subws.Route(subws.GET(rest.ResourcePath(subresourcesvmiGVR) + rest.SubResourcePath("test")).
-			To(func(request *restful.Request, response *restful.Response) {
-				response.WriteHeader(http.StatusOK)
-			}).
+			To(func(request *restful.Request, response *restful.Response) {}).
```

Comment:
At this place I find it better readable to explicitly set the OK status... but that might be a matter of taste

Reply:
I agree, that's why I added the comment above, do you think it's worth putting it back, I don't mind either way after I read your explanation (which is great, thanks!).

Reply:
ah I missed the comment. Works for me as it is then :)